tests/telemetry/test_telemetry.py

```diff
@@ -129,6 +136,7 @@ def test_telemetry_ouput(duthosts, enum_rand_one_per_hwsku_hostname, ptfhost,
 
 
 @pytest.mark.parametrize('setup_streaming_telemetry', [False], indirect=True)
+@pytest.mark.disable_loganalyzer
```

Comment:
>disable_loganalyzer [](http://example.com/codeflow?start=13&length=19)

this is not explained in PR description. why it is needed?

Reply:
Added in PR description